### PR TITLE
Dont repeat menus

### PR DESCRIPTION
**Pull request by shemgp**
If menus are generating existing menus, then don't add them anymore.

### CODE REVIEW

File: src/Generators/Scaffold/MenuGenerator.php [hunk 1]
```diff
@@ -27,12 +27,18 @@ public function __construct(CommandData $commandData)
     {
         $this->commandData = $commandData;
         $this->path = config(
-            'infyom.laravel_generator.path.views',
+            'infyom.laravel_generator.path.menu_views',
```
Comment (mitulgolakiya) [inline src/Generators/Scaffold/MenuGenerator.php]: we do not have such configuration.

File: src/Generators/Scaffold/MenuGenerator.php [hunk 2]
```diff
@@ -27,12 +27,18 @@ public function __construct(CommandData $commandData)
...
             base_path('resources/views/'
             )
         ).$commandData->getAddOn('menu.menu_file');
         $this->templateType = config('infyom.laravel_generator.templates', 'adminlte-templates');
 
+        if (!is_dir(dirname($this->path))) {
```
Comment (mitulgolakiya) [inline src/Generators/Scaffold/MenuGenerator.php]: probably should use FileUtil class.

**Comment by mitulgolakiya** [comment]
@shemgp requested some changes.

**Comment by ajayinfyom** [comment]
thanx for your contribution.
we fixed this issue on another PR. so closing this one.

**Pull request closed without merging**